### ISSUE TITLE
Tethys Developer Version Installation 'staticfiles' is not a registered tag library

### ISSUE THREAD

**Issue by romer8** [tethysplatform/tethys#793]
I have installed the version the development version using miniconda with the following command : 
```bash
conda create -n tethys -c tethysplatform/label/dev -c tethysplatform -c conda-forge tethys-platform
tethys gen portal_config
tethys db configure
``` 
This installation comes with python 3.10.4 However, when I start the Tethys with the command `tethys manage start` I get the following error:

```bash

(tethys) [gio@gio tethys]$ tethys manage start
Loading Tethys Extensions...
Loading Tethys Apps...
Loading Tethys Extensions...
Loading Tethys Apps...
Performing system checks...

System check identified no issues (0 silenced).
April 20, 2022 - 21:01:14
Django version 3.2.12, using settings 'tethys_portal.settings'
Starting ASGI/Channels version 3.0.4 development server at http://127.0.0.1:8000/
Quit the server with CONTROL-C.
ERROR:django.request:Internal Server Error: /
Traceback (most recent call last):
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/defaulttags.py", line 1037, in find_library
    return parser.libraries[name]
KeyError: 'staticfiles'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/asgiref/sync.py", line 451, in thread_handler
    raise exc_info[1]
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/core/handlers/exception.py", line 38, in inner
    response = await get_response(request)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/core/handlers/base.py", line 233, in _get_response_async
    response = await wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/asgiref/sync.py", line 414, in __call__
    ret = await asyncio.wait_for(future, timeout=None)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/asyncio/tasks.py", line 408, in wait_for
    return await fut
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/asgiref/current_thread_executor.py", line 22, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/asgiref/sync.py", line 455, in thread_handler
    return func(*args, **kwargs)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/tethys_portal/views/home.py", line 29, in home
    return render(request, template, {"ENABLE_OPEN_SIGNUP": settings.ENABLE_OPEN_SIGNUP,
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/base.py", line 170, in render
    return self._render(context)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/base.py", line 162, in _render
    return self.nodelist.render(context)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/loader_tags.py", line 150, in render
    return compiled_parent._render(context)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/base.py", line 162, in _render
    return self.nodelist.render(context)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/loader_tags.py", line 183, in render
    template = context.template.engine.select_template(template_name)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/engine.py", line 174, in select_template
    return self.get_template(template_name)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/engine.py", line 143, in get_template
    template, origin = self.find_template(template_name)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/loaders/base.py", line 29, in get_template
    return Template(
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/base.py", line 155, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/base.py", line 193, in compile_nodelist
    return parser.parse()
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/defaulttags.py", line 1088, in load
    lib = find_library(parser, name)
  File "/home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/django/template/defaulttags.py", line 1039, in find_library
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'staticfiles' is not a registered tag library. Must be one of:
admin_list
admin_modify
admin_urls
analytical
cache
chartbeat
clickmap
clicky
crazy_egg
django_bootstrap5
facebook_pixel
gauges
google_analytics
google_analytics_js
gosquared
gravatar
guardian_tags
hotjar
hubspot
humanize
i18n
intercom
kiss_insights
kiss_metrics
l10n
log
mixpanel
olark
optimizely
performable
piwik
rating_mailru
recaptcha2
rest_framework
session_security_tags
site_settings
snapengage
spring_metrics
static
tags
terms_tags
tethys_gizmos
tethys_services
tz
uservoice
woopra
yandex_metrica
ERROR:django.channels.server:HTTP GET / 500 [0.07, 127.0.0.1:38068]
```
I am able to fix this by editing the file at In template /home/gio/miniconda3/envs/tethys/lib/python3.10/site-packages/session_security/templates/session_security/all.html, error at line containing `{% load static from staticfiles %}`

```html
 {% comment %}
 This demonstrates how to setup session security client side stuff on your own.
 It provides sensible defaults so you could start with just::
 {% include 'session_security/all.html' %}
{% endcomment %} 
 {% load session_security_tags %}
{% load i18n l10n %}
{% load static from staticfiles %}
 
{# If the user is not authenticated then there is no session to secure ! #}
{% if request.user.is_authenticated %}
 
{# The modal dialog stylesheet, it's pretty light so it should be easy to hack #}
<link rel="stylesheet" type="text/css" href="{% static 'session_security/style.css' %}">
 
{# Include the template that actually contains the modal dialog #}
{% include 'session_security/dialog.html' %}
```
I would like to know if there is another fix besides editing the file directly

**Comment by swainn** [comment]
I think this is actually fixed in django-sessionsecurity 2.6.6. You need to list the conda-forge channel first to install the correct version:

```
conda create -n tethys4 -c conda-forge -c tethysplatform/label/dev tethys-platform
```

**Comment by swainn** [comment]
UPDATE: I've removed the outdated packages on `tethysplatform/label/dev` so now it shouldn't matter what order you list the channels.
